dynamic/Dockerfile

```diff
@@ -107,7 +107,8 @@ ADD help.1 /help.1
 COPY --from=instana-agent /opt/instana /opt/instana
 COPY --from=instana-agent /usr/lib/tmpfiles.d/instana-agent.conf /usr/lib/tmpfiles.d/instana-agent.conf
 
-RUN ln -s /opt/instana/agent/licenses /licenses
+RUN mkdir /licenses \
+    && cp -r /opt/instana/agent/licenses/* /licenses/
```

Comment:
Then rather not copy over the license folder in the first place, this doubles the files

Reply:
I agree, if it needs to be in /licenses then only have it in /licenses